cmd/system-probe/api/module/loader.go

```diff
@@ -158,8 +158,10 @@ func Close() {
 func updateStats() {
 	start := time.Now()
 	then := time.Now()
+	now := time.Now()
 	ticker := time.NewTicker(15 * time.Second)
-	for now := range ticker.C {
+
+	for {
```

Comment:
Will we ever exit this loop? Maybe use a `select`?

Reply:
I'm not sure I follow, how would a `select` improve uponn this?  This loop never exits, and changing it to do so would be a much larger change, as we'd need to manage shutdown of modules.

Reply:
Yeah there is no shutdown process for `updateStats`. I guess you could exit the loop in the old code by closing the channel, which you can't do here. However, apparently you can't close `ticker.C` since it is a receive only channel. So the difference between the new and old code is moot.